platform/ios/src/MGLMapView.mm

```diff
@@ -4156,7 +4151,12 @@ - (MGLAnnotationTag)annotationTagAtPoint:(CGPoint)point persistingResults:(BOOL)
             {
                 if ([annotation isKindOfClass:[MGLMultiPoint class]])
                 {
-                    return false;
+                    if ([self.delegate respondsToSelector:@selector(mapView:shapeAnnotationIsEnabled:)] &&
```

Comment:
Not sure if it matters for performance reasons, but for better readability, you could factor this call out of the `std::remove_if()` call. (×2)